src/index.js

```diff
@@ -0,0 +1,73 @@
+const fs = require('fs');
+const { join } = require('path');
+
+class StaticAssetsManager {
+  constructor({
+    staticAssetVersions = {},
+    useStaticAssetVersions = false,
+    staticAssetDirLocal = '/public',
+    staticAssetDirCDN = '/public',
+  }) {
+    this.staticAssetVersions = staticAssetVersions;
+    this.useStaticAssetVersions = useStaticAssetVersions;
```

Comment:
Since this is set via `setStaticAssetVersions` I don't know if it needs to be exposed as an option for the class. Same with `staticAssetVersions`, when would that be passed directly to the class constructor since it's not exposed outside of this file? What do you think?

Reply:
We could potentially make these arg names less verbose too since they're all within the static asset manager

Reply:
@djfarrelly great point! I don't recall well why I took this approach. I think what may have drove this is to make it useful for anyone as it is an open-source project. Let's simplify things 👍